src/components/Review/ReviewSectionRowAction.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Grid } from 'semantic-ui-react'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { ReviewSectionComponentProps } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const ReviewSectionRowAction: React.FC<ReviewSectionComponentProps> = ({
+  action,
+  section: { details },
+  isAssignedToCurrentUser,
+  thisReview,
+}) => {
+  const {
+    location: { pathname },
+  } = useRouter()
+
+  const reviewPath = `${pathname}/${thisReview?.id}`
```

Comment:
Possible problem if `thisReview` non existent?

Reply:
hmm, it's only really used in the use 'cases' below where review does exist, so I thought it's fine to define it once on top (rather then in each case